src/Components/__stories__/StaticCollapse.story.tsx

```diff
@@ -30,6 +30,20 @@ class CollapseStory extends React.Component {
               </Collapse>
             </BorderBox>
           </Flex>
+          <Spacer mb={4} />
+          <Flex alignItems="flex-start">
+            <BorderBox width="300px" flexDirection="column">
+              <StaticCollapse open={!this.state.open}>
+                <Box>Hello I am some content inside StaticCollapse</Box>
+              </StaticCollapse>
+            </BorderBox>
+            <Spacer ml={4} />
+            <BorderBox width="300px" flexDirection="column">
```

Comment:
@ds300 - this was indeed a configuration issue. Reading here: https://react-spring.surge.sh/gotchas#animating-auto we just needed to give our existing `Collapse` component more information so that the positioning and size could be inferred: 

```js
<BorderBox width="300px" flexDirection="column" position="relative">
  <Collapse open={this.state.open}>
    <Box>Hello I am some content inside regular old Collapse</Box>
  </Collapse>
</BorderBox>
```

`position='relative'` on the container did the trick.

Reply:
Sweet!!! 🙌 Thanks so much for figuring it out! Do you mind getting rid of the StaticCollapse component?

Reply:
Oh like push to this branch? Sure thing.

Reply:
@ds300 - I know your OOO but heads up I pushed to here: https://github.com/artsy/reaction/pull/1984 -- cc @sweir27 

If anyone knows exact steps on how to commit directly into this PR i can close that other one.

Reply:
you should be able to push to other people's PRs in the artsy org if you add their repo as a remote via ssh and then push to the PR branch

Reply:
Oh nice, not so bad (done)